server/adaptors/integrations/__data__/repository/aws_cloudtrail/assets/create_table_cloud-trail-1.0.0.sql

```diff
@@ -56,10 +55,10 @@ CREATE EXTERNAL TABLE IF NOT EXISTS {table_name} (
       cipherSuite:STRING,
       clientProvidedHostHeader:STRING
     >
-  >>
-) USING json
-LOCATION '{s3_bucket_location}'
+)
+USING json
 OPTIONS (
-  compression='gzip',
-  recursivefilelookup='true'
+   PATH '{s3_bucket_location}',
+   recursivefilelookup='true',
+   multiline 'true'
```

Comment:
Is multiline part case sensitive. I've usually seen commands with `multiLine` L captialised.

Reply:
```
PATH='{s3_bucket_location}',
recursivefilelookup='true',
multiLine='true'
```

Reply:
I think both `multiline` and `multiLine` are be accepted